### DISCUSSION TITLE
GP-LOG Colors

### DISCUSSION THREAD

**Discussion by lyricsdude** [gopro/labs#626] [General]
I own the Gopro 11 and not the Gopro 12 (yet). Are the GP-LOG Colors similar to the flat colors on my Gopro 11? I can't find any info about this.

I see there's a creative LUT for the Gopro 12 firmware version 1.20. But if I use it on my Gopro 11 flat footage it's just a gamma curve correction.

**Comment by dnewman-gpsw** [comment]
I've look at the LUTs and they are more than gamma correction.  That are designed to make GP-Log look like GoPro Natural, and I've found that they do this well.

Use cFoMLOGB=400 on HERO11 for the LUTs to work.
